src/poprox_recommender/handler.py

```diff
@@ -1,20 +1,32 @@
 import json
+import logging
 
 from poprox_concepts.api.recommendations import (
     RecommendationRequest,
     RecommendationResponse,
 )
 from poprox_recommender.default import select_articles
 
+logger = logging.getLogger(__name__)
+logger.setLevel(logging.DEBUG)
```

Comment:
I'd rather see this in a different logging configuration place, but cleaning up log config can be a separate PR (which I was thinking of doing anyway, because I was looking for logging while debugging #19).

Reply:
Yeah, no strong opinions about how logging ought to be set up. Mostly haven't worried about it at all thus far but needed a way to make it visible which params were getting received in order to know this was working